packages/kotti-ui/source/kotti-field-inline-edit/types.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod'
+
+import { KottiField } from '../kotti-field/types'
+
+export namespace KottiFieldInlineEdit {
+	export const valueSchema = z.string().nullable()
+	export type Value = z.output<typeof valueSchema>
+
+	export enum ConfirmationValidation {
+		ERROR = 'error',
+		WARNING = 'warning',
+		NEVER = 'never',
+	}
+
+	export enum TextStyle {
+		HEADER_ONE = 'h1',
+		HEADER_TWO = 'h2',
+		HEADER_THREE = 'h3',
+	}
+
+	export const propsSchema = KottiField.propsSchema
+		.merge(KottiField.potentiallySupportedPropsSchema.pick({ tabIndex: true }))
+		.extend({
+			isReadonly: z.boolean().default(false),
```

Comment:
Could be preference but shouldn't this be `isReadOnly`?

Reply:
I would only capitalize the o if the read-only were two separate words, usually the camel casing follows to separate words and in that case `readonly` should be one, thus `isReadonly`

seem goes for `multiline`